<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkDialog</class>
 <widget class="QDialog" name="WorkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QGroupBox" name="statuxBox">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Status</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>118</width>
          <height>185</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="openBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Open</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/res/icons/active.svg</normaloff>:/res/icons/active.svg</iconset>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statusGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="doneBtn">
           <property name="text">
            <string>Done</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/res/icons/done.svg</normaloff>:/res/icons/done.svg</iconset>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statusGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="interruptedBtn">
           <property name="text">
            <string>Interrupted</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/res/icons/interrupted.svg</normaloff>:/res/icons/interrupted.svg</iconset>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statusGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="heldBackBtn">
           <property name="text">
            <string>Held Back</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/res/icons/heldback.svg</normaloff>:/res/icons/heldback.svg</iconset>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statusGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="freeBtn">
           <property name="text">
            <string>Free</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/res/icons/free.svg</normaloff>:/res/icons/free.svg</iconset>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statusGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="approvedBtn">
           <property name="text">
            <string>Approved</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/res/icons/approved.svg</normaloff>:/res/icons/approved.svg</iconset>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statusGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="invoicedBtn">
           <property name="text">
            <string>Invoiced</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/res/icons/invoiced.svg</normaloff>:/res/icons/invoiced.svg</iconset>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">statusGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="fromDateTime">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="toDateTime">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="pauseEdit">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="noteEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WorkDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WorkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="statusGroup"/>
 </buttongroups>
</ui>
